aws/README.md

```diff
@@ -4,12 +4,30 @@ Following the pattern laid out in "[How to create reusable infrastructure with T
 
 ## Getting Started
 
+### Get an AWS account
+
+Ask the Ops team to set up an account and train you. (The training doc is [here](../USER-TRAINING.md) if you're curious.)
+
+### Enable Multi-Factor Authentication (MFA) on your AWS account
+
+I like [Duo](https://duo.com/product/trusted-users/two-factor-authentication/duo-mobile), but any tool from [Amazon's list](https://aws.amazon.com/iam/details/mfa/) should be fine.
+
+* From the AWS Dashboard, navigate to IAM controls (under Services or use the search box)
+   * Users
+   * Select your IAM
+   * Select "Security credentials"
+   * Edit "Assigned MFA device"
+   * Choose "Virtual MFA device" (unless you have a dedicate hardware device such as a Yubikey)
+   * Follow prompts to register MFA device
+
```

Comment:
I didn't know Duo app, seems that have a very good feature: BACKUPS, Google Authenticator doesn't have it. Other way to have backups is to keep the QR code in a **very safe place**, just in the case you break or lost your Virtual MFA device.